requirements.txt

```diff
@@ -1,23 +1,23 @@
-apache-airflow==1.10.14 # pyup: ignore
+apache-airflow==2.6.0 # pyup: ignore
```

Comment:
Does this requirement mean that this tool will be used only for Airflow 2.6.0?
If so, can we have e.g. some range here?

Reply:
No, this tool uses some utils functions and AirflowExeception from the airflow package. On conversion this package nohow influences and users could use this tool for all Airflow 2 sub-verision.

Reply:
ok, sg